apps/aecore/test/aec_mining_tests.erl

```diff
@@ -31,10 +31,12 @@ mine_block_test_() ->
          fun() ->
                  TopBlock = #block{height = 0,
                                    target = ?HIGHEST_TARGET_SCI},
-                 meck:expect(aec_pow, pick_nonce, 0, 9),
```

Comment:
I don't see how this is related to this PR.

If the behavior for this test case changed due to this PR, I think we should find the reason for this.

If this is a general test case improvement, make another PR.

Reply:
The reason was that crypto adds salt and therefore has a non-deterministic result on computing the signed tx. I did not want to hard-code the signed_tx, but realize that I have to.